code/src/index.css

```diff
@@ -11,3 +11,96 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* Mobile-first  */
+.flex-parent {
+  background-color: pink;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.message-display-div {
+  background-color: white;
+  border: 2px solid grey;
+  box-shadow: 8px 7px 5px black;
+  word-wrap: break-word;
+  width: 80vw;
+  margin: 8px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.message-display-div span {
+  display: flex;
+  justify-content: space-between;
+}
+
+.form-div {
+  background-color: white;
+  border: 2px solid grey;
+  box-shadow: 8px 7px 5px black;
+  margin: 30px;
+  width: 80vw;
+  height: 80vh;
+  padding-left: 20px;
+  padding-right: 20px;
+  text-align: center;
+}
+
+.form-div input {
+  width: 80vw;
+  height: 50px;
+  font-size: 20px;
+}
+
+.form-button {
+  background-color: pink;
+  border: none;
+  border-radius: 50px;
+  font-size: 20px;
+  margin-top: 30px;
+}
+
+@media screen and (min-width: 768px) {
+  .message-display-div {
+    width: 30vw;
+  }
+  
+  .form-div {
+    margin: 30px;
+    width: 30vw;
+    height: 70vh;
+  }
+  
+  .form-div input {
+    width: 30vw;
+    height: 30px;
+  }
+
+  .form-button {
+    font-size: 15px;
+    margin-top: 10px;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .form-div {
+    margin: 30px;
+    width: 30vw;
+    height: 60vh;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .form-div {
+    margin: 30px;
```

Comment:
Same here with line 97 and 98, same as line 71 and 72. I'm not sure if we remove it, things will stay the same (I think it should).